### DISCUSSION TITLE
About the "disable User Agent client hints" patch.

### DISCUSSION THREAD

**Discussion by bat999** [bromite/bromite#1307] [Q&A]
Hi
Changelog for 92.0.4515.103 shows "disable User Agent client hints".
And I can see the patch here ---> 
**https://github.com/bromite/bromite/blob/master/build/patches/Disable-UA-client-hint.patch**
But Bromite is still sending out the User Agent Client Hints.
Is there some additional setting or a flag that is needed to make the patch work?

**https://user-agent-client-hints.glitch.me/**

![client_hints](https://user-images.githubusercontent.com/640761/127195760-200c52c5-43e1-43db-a103-1e86a211aa0c.jpg)

**Comment by csagan5** [comment]
Please submit a bug for this; discussions are not for bug reports or feature requests.
